src/cli.js

```diff
@@ -13,6 +13,7 @@ prog
 	.option('--entry, -i', 'Entry module(s)')
 	.option('--output, -o', 'Directory to place build files into')
 	.option('--format, -f', 'Only build specified formats', 'es,cjs,umd')
+	.option('--target', 'Specify your target environment', 'node')
```

Comment:
I use `--target`, because that's [the name in webpack](https://webpack.js.org/configuration/target/). However, `--environment` or `--platform` may be more obvious names.

Reply:
I'm good with replicating webpack's naming there.